dawn/src/dawn/CodeGen/Cuda-ico/ASTStencilBody.cpp

```diff
@@ -114,7 +114,9 @@ std::string ASTStencilBody::makeIndexString(const std::shared_ptr<iir::FieldAcce
   bool isSparse = unstrDims.isSparse();
 
```

Comment:
`std::string denseSize = ...` is repeated below. It's safe to move it here, as the `isVertical` case is already excluded at this point.

Reply:
good observation!